src/components/EditMenu.vue

```diff
@@ -442,6 +442,66 @@
           </div>
         </template>
       </ExpansiblePanel>
+      <ExpansiblePanel
+        v-for="miniWidgetContainer in miniWidgetsBars"
+        :key="miniWidgetContainer.name"
+        :compact="interfaceStore.isLg || interfaceStore.isOnSmallScreen ? true : false"
+        invert-chevron
+        hover-effect
+        elevation-effect
+        no-top-divider
+        :is-expanded="miniWidgetContainer?.widgets!.length > 0"
+      >
+        <template #title>
+          <div
+            class="flex w-[90%] justify-between items-center 2xl:text-[18px] xl:text-[16px] lg:text-[14px] -mb-3 font-normal ml-2"
+          >
+            {{ miniWidgetContainer.name }}
+            <v-badge
+              :content="miniWidgetContainer.widgets?.length"
+              color="#3B78A8"
```

Comment:
```suggestion
              color="#4FA483"
```
Consistent accent colour, following on from #1606

Reply:
done!

P.S.: I like the kelp green accent color, but I’m concerned about its contrast with the white text.
On my monitor, it’s a little hard to read, especially in very bright environments.
Does it look the same to you?

How about making the color a little darker while keeping the same tone?

Reply:
I'll ask SMS if we can change the colour, either everywhere or specifically when using it in an interface. Probably best to merge as-is for now, and we can raise an Issue to track the idea of changing it :-)

It would also be nice for colours to be defined as variables rather than hardcoded everywhere, so that might make sense as part of a broader issue 🤷‍♂️ :-)

Reply:
Raised in #1611 - out of scope for this PR :-)